karma.saucelabs.conf.js

```diff
@@ -33,7 +33,7 @@ module.exports = config => {
       platformName: "Windows 11",
       browserVersion: "latest",
       "sauce:options": {
-        geckodriverVersion: "0.27.0"
+        geckodriverVersion: "0.34.0"
```

Comment:
Firefox unit tests were failing because of a bug in the driver. I need to force a certain version that contains the fix.